setup.py

```diff
@@ -6,19 +6,24 @@
 
 tests_require = [
     'ftw.builder',
+    'ftw.simplelayout [contenttypes]',
     'plone.app.testing',
-    'ftw.testbrowser'
+    'ftw.testbrowser',
+    'ftw.contacts [zip_export]',
+    ]
+zip_export = [
+    'ftw.zipexport'
     ]
-
 extras_require = {
-    'tests': tests_require
+    'tests': tests_require,
+    'zip_export': zip_export
```

Comment:
Maybe also add a `simplelayout` extras as we have specific simplelayout integration?

And also add it to the development.cfg as it makes development easier.
E.g.:

```ini
[instance]
eggs += ftw.contacts [simplelayout]
```